pkg/codegen/dotnet/gen.go

```diff
@@ -1299,9 +1308,6 @@ func (mod *modContext) genFunctionFileCode(f *schema.Function) (string, error) {
 	for _, i := range imports {
 		importStrings = append(importStrings, i.SortedValues()...)
 	}
-	if f.NeedsOutputVersion() {
-		importStrings = append(importStrings, "Pulumi.Utilities")
```

Comment:
Is this not needed anymore? Is that related to this change?

Reply:
It became unnecessary, I'm not sure when. It's not related to the change.